#goal definition
geometry_msgs/PoseStamped[] goals
int32[] targetIds
int32[] taskIds
string taskType
---
#result definition
bool isCompleted
string taskType
int32[] taskIds
string description
int32 robotId
---
 #feedback
int32 robotId
int32 doorId 
time measureTime
uint8 doorStatus
